### ISSUE TITLE
Add a "Cleared" transaction status

### ISSUE THREAD

**Issue by Snaptags** [moneymanagerex/moneymanagerex#1405]
ac0935p suggested:
> For credit card transactions (amongst others) it would be great to have a "Cleared" status between "None" and "Reconciled". Typically a transaction is entered on the day it is made, but not reconciled until the monthly statement is issued. In between those two events, transactions do appear on the online account and it is nice to be able to track those that have been listed and those that haven't yet.

> I am a long-time user of Financisto, moving to MMX and this is the only feature I miss, so far.

I'd like to second this improvement, because after migrating from anMoney I miss that state, too.

[has to be coordinated with the corresponding Android issue](https://github.com/moneymanagerex/android-money-manager-ex/issues/1091)

**Comment by lexa2** [comment]
Hi. My approach for tracking this is to use "None" status for transactions that are not yet shown in internet-bank and "Followup" for transactions that are shown in internet-bank as a "Funds blocked for the authorized transaction but confirming docs are not yet received by the bank".

**Comment by Snaptags** [comment]
Well, the documentation suggests "follow up" being a state where additional action is required:

> This status marks transactions as needing more action. For example, you receive a balance statement from the financial institution and you notice that the transaction amount is different between what you recorded and what is in the statement. You can mark it as flag for follow up so that you can followup with the financial institution.

So one **could** use the state as a replacement for "cleared", but then there is no state left to mark transactions as "to do" :-(

**Comment by lexa2** [comment]
For "to do" you should use "recurring transactions" function. Hint: repeating is not a requirement for a recurring transaction, it is perfectly ok to have a "one time" recurring transaction. Maybe terminology used by MMEx for this function is a bit misleading, close analog for it is a "Calendar scheduler" for meetings/appointments.

**Comment by StevenRenich** [comment]
I agree that this feature would be very useful. It would go a long way to making the reconcile feature easier and more like some of the commercial money software.

A quick search of "reconcile" on the MMEX forum shows several posts where people are switching (or considering) to MMEX from MS Money, Quicken, etc. (I switched last year). The reconcile features on such software attempts to reconcile a monthly bank statement by: 
 1) Marking all matching transactions as "cleared"
 2) Verifying that the balance of the cleared and (previously) reconciled transactions matches the new balance on the statement.
 3) Accept the changes by changing all cleared transactions to reconciled.

Note that the "cleared" feature alone would enable people to reconcile account statements in a manner conceptually the same as the software some of us are switching from.

**Comment by alanwren** [comment]
+1 to this feature request.  I currently use the same approach as lexa2 but have always thought that this would be nice to have.
